src/lib/BikeHopperClient.ts

```diff
@@ -39,6 +39,27 @@ export async function fetchRegionConfig(): Promise<RegionConfig> {
   return RegionConfigSchema.parse(await result.json());
 }
 
+function timeStampToLocalTime(timeStamp: number) {
+  const sysDate = new Date();
+  const sysOffset = sysDate.getTimezoneOffset();
+
+  const pst_formatter = new Intl.DateTimeFormat('en-US', {
```

Comment:
```suggestion
  const pstFormatter = new Intl.DateTimeFormat('en-US', {
```
Camel case is the usual style in JS